app/models/User.class.js

```diff
@@ -3,6 +3,7 @@
 const _ = require('lodash')
 const ServiceRole = require('./ServiceRole.class')
 const { isInternalGDSEmail } = require('../utils/email-tools')
+const DEGATEWAY_FLAG = process.env.DEGATEWAY_FLAG === 'true'
```

Comment:
I think this could be better named - I assume this is the global enable/disable option? I think something like `ENABLE_DEGATEWAY_VIEW` would convey the purpose a bit better